src/main/java/com/commercetools/sync/products/utils/ProductReferenceResolutionUtils.java

```diff
@@ -247,31 +244,21 @@ static CategoryReferencePair mapToCategoryReferencePair(@Nonnull final Product p
    *     references expanded.
    */
   @Nonnull
-  public static ProductQuery buildProductQuery() {
-    return ProductQuery.of()
+  public static ProductProjectionQuery buildProductQuery() {
```

Comment:
Those methods will be changed by the other PR. Not sure how we will be manage the conflicts, probably this PR needs to be merged first in a release branch, not to master.

Reply:
I have now change the base of this pr to branch 'release_5_0_0'